### ISSUE TITLE
Combobox: list options are not announced with JAWS and user will not know which item to select

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#4027]
**Describe the bug**
When user select one option from the combobox with keyboard, the selected option cannot be announced sometime by JAWS user. Which will blocking the JAWS users. 

**Isolated Example**
https://codesandbox.io/s/c2fj5?file=/package.json:157-278

**To Reproduce**
1. Go to https://c2fj5.csb.app/
2. Open JAWS
2. Tab to focus the control
3. Using down - narrow key to select the item from top to bottom
4. Selected item cannot be announced sometimes

**Expected behavior**
Selected item should be announced always when selected

**Screenshots**
https://user-images.githubusercontent.com/5724592/135390055-4ec17669-38ad-42cd-853d-d33b8031e1c1.mp4

**UI5 Web Components for React Information**
  "@ui5/webcomponents": "0.31.19",
   "@ui5/webcomponents-fiori": "0.31.19",
   "@ui5/webcomponents-react": "0.131.20",
Operating System: Window 10
Browser: Chrome

**Additional context**
I am from Successfactors, currently, it is a blocking issue.

**Comment by Todor-ads** [comment]
Hi @sleric1024,
Thank you for sharing this finding.
This issue has been fixed in the latest version of Combo Box.
Regards,
Todor

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rl according to the @Todor-ads comment it seems the issue has been fixed in the master. However, the author is using 0.31 where the fix is not present. I checked the commit history for fixes that can be related to the issue and I found this one https://github.com/SAP/ui5-webcomponents/pull/3358 that seems to be the one. 

Could you go trough the issue description and confirm that https://github.com/SAP/ui5-webcomponents/pull/3358  should be the fix?



Steps to reproduce the behavior:
- Go to https://c2fj5.csb.app/
- Start JAWS
- Tab to focus the control
- Using arrow-down key to select the item from top to bottom
- **Selected item cannot be announced sometimes**

**Comment by elenastoyanovaa** [comment]
Hello @sleric1024,

Can you please add some more information: Is the picker opened, when you navigate, or it is closed? Did you re-test to confirm that the issue is not reproducible in the latest version? This will help us find the right PR, which fixed the issue :)

Kind Regards,
Elena

**Comment by sleric1024** [comment]
Hi @elenastoyanovaa ,
I made a video record on https://user-images.githubusercontent.com/5724592/135390055-4ec17669-38ad-42cd-853d-d33b8031e1c1.mp4, you may refer to this link for detail information. And  this issue is not fixed yet in version **0.31.20**. As for the latest version 0.31.22 (2021-10-26). I will check on my colleague's laptop (Only happened in **Windows with JAWS**) tomorrow. And I will replay you soon.

Best Regards,
Eric

**Comment by elenastoyanovaa** [comment]
Hello @sleric1024 ,

The issue is not related to the component, but it is due to the announcement queue present in screen readers. Screen readers use a queue to collect speech outputs that need to be announced and some actions may interrupt the current speech output and information could be missed. There is nothing different in regards to the element structure between the 2nd and 3rd item, the difference comes from the testing. You are pressing ARROW DOWN while Jaws is still announcing the information regarding the next item and this causes the update of the combobox value to not be announced. For example, in a real scenario, the screen reader user would wait until the information regarding the currently selected item is announced and then navigate to the next one. If you test it yourself you will see that all items are announced correctly, when not performing actions, while JAWS is announcing. 

The fix which was mentioned in the comments above is applied to the used version, but do not affect this behavior, since it is not implementation specific. As the missing announcement comes from the value update of an input field, not by some live region or additional aria attributes added by us, this is prioritised by the screen reader and we could not manipulate it.

Kind Regards,
Elena
